PWGJE/EMCALJetTasks/Tracks/AliAnalysisTaskEmcalSoftDropResponse.cxx

```diff
@@ -250,7 +361,7 @@ std::vector<double> AliAnalysisTaskEmcalSoftDropResponse::MakeSoftdrop(const Ali
   if(tracks && (fUseChargedConstituents || isMC)){                    // Neutral particles part of particle container in case of MC
     AliDebugStream(1) << "Jet substructure: Using charged constituents" << std::endl;
     for(int itrk = 0; itrk < jet.GetNumberOfTracks(); itrk++){
-      auto track = jet.TrackAt(itrk, tracks->GetArray());
+      auto track = jet.Track(itrk);
```

Comment:
To be check whether this works fine, also with the embedding framework. @raymondEhlers ?

Reply:
@mfasDa `jet.TrackAt(itrk, tracks->GetArray())` was deprecated when the new embedding framework was introduced. `jet.Track(itrk)` doesn't need the TClonesArray and will always return the right track regardless of whether one is using the embedding or not.

(Note that `jet.TrackAt(itrk, tracks->GetArray())` will still also always give the right track, but it will aggressively throw warnings when embedding)